plugins/auth-backend/src/identity/router.ts

```diff
@@ -34,7 +34,21 @@ export function createOidcRouter(options: Options) {
     jwks_uri: `${baseUrl}/.well-known/jwks.json`,
     response_types_supported: ['id_token'],
     subject_types_supported: ['public'],
-    id_token_signing_alg_values_supported: ['RS256'],
+    id_token_signing_alg_values_supported: [
+      'RS256',
+      'RS384',
+      'RS512',
+      'ES256',
+      'ES384',
+      'ES512',
+      'PS256',
+      'PS384',
+      'PS512',
+      'HS256',
```

Comment:
This endpoint does publish the certificates for service tokens, and symmetric signing will never be used for user tokens, so we can skip these

Reply:
Updated in 422c93dd7c98452270aa4527cd7b024f85dca2e9